Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.6
+FROM alpine:3.7
```

Comment:
why this change?

Reply:
See commit comment. Alpine 3.7 brings go 1.9.4 with `json.Valid` function (afaik it's not available in Go version that was default in Alpine 3.6). But maybe there is other solution for that? Why it's important to stick with 3.6 anyway?